app/frontend/src/pages/ask/Ask.tsx

```diff
@@ -259,6 +279,30 @@ export function Component(): JSX.Element {
                     snapToStep
                 />
 
+                <Slider
+                    className={styles.chatSettingsSeparator}
+                    label="Min Search Score"
+                    min={0}
+                    max={1}
+                    step={0.1}
+                    defaultValue={minSearchScore}
+                    onChange={onMinSearchScoreChange}
+                    showValue
+                    snapToStep
+                />
+
+                <Slider
+                    className={styles.chatSettingsSeparator}
+                    label="Min Reranker Score"
```

Comment:
```suggestion
                    label="Minimum reranker score"
```

...if we have room for "minimum", I think we probably do?